t/rest_server/contents/main.go

```diff
@@ -94,8 +95,8 @@ func main() {
 
 	flag.Parse()
 	if err := realMain(logging.WithLogger(ctx, logger)); err != nil {
-		done() // deferred function calls won't execute due to logger.Fatal() never returning
-		logger.Fatal(err)
```

Comment:
Shouldn't we log this error?

Reply:
github.com/abcxyz/pkg/logging's logger doesn't have a Fatal. What should I do?

Reply:
Well, the behavior of Fatal was to log then exit, so I think you can log at the most severe level available. And there's already a call to Exit() below.

Reply:
Ok, I was going to do logging.LevelEmergency but I think LevelError sounded more appropriate.

Reply:
I think logger.ErrorContext() is more conventional than logger.Log(..., logging.LevelError)

Reply:
Got it, thank you!